modules/ROOT/pages/home-page/index.adoc

```diff
@@ -1,6 +1,15 @@
 = Home page
 :description: This section describes the Home Page of the Neo4j Ops Manager.
 
+Once running, Ops Manager can, by default, be accessed by navigating to the following location:
+
+----
+http(s)://<server>:8080
```

Comment:
```suggestion
https://<server>:8080
```

We expose HTTPS by default, do we? If yes, customers would need to make additional config to have 8080 over HTTP instead. If they do so, they'll know where to go I think. So maybe to avoid confusion we could go for HTTPS here. You also write "by default" above.

Reply:
I think we do both, but https is likely to present with a cert error the first time? Could be wrong though.

Reply:
I was wrong 😄